src/redux/books/BookSlice.js

```diff
@@ -1,36 +1,54 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const postData = createAsyncThunk('data/postData', async () => {
+  try {
+    const response = await axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/');
+    console.log(response.data);
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+export const getListBook = createAsyncThunk('data/getListBook', async () => {
```

Comment:
<img width="972" alt="1" src="https://github.com/massazoum/Zoum_Bookstore/assets/77843050/47aae80b-5480-4b21-bc72-f5e58d361c4a">

- [ ] The image's requirement said it all clearly, please create an action called fetchbooks, I think it does matter to follow the need and update the action name everywhere in the app